unfetter-discover/unfetter-ui
# Issue 840

This fully implements an Overall Rating chart for creating/editing an assessment.
It also increases test coverage for the assessment wizard component by about 40%.

Overall Rating chart should give average risk rating for an assessment by averaging the risk for each phase in the assessment type (indicator, mitigation, or sensor) when there is one assessment type.
If there are multiple assessment types, it should show an average risk by assessment type where assessment type risk is calculated as above.

To Test:
Create an assessment with Sensor type. See that the Overall chart matches the average risk for each phase as you manipulate the phase ratings.
Create an assessment with all three types. See that the Overall chart matches the average risk for all assessment types as you manipulate the phase ratings.
